### ISSUE TITLE
re-create ingress after removal

### ISSUE THREAD

**Issue by o-grigorev** [TremoloSecurity/openunison-k8s-operator#44]
Ingress object is not re-created by openunison-operator if it was deleted. 
https://github.com/TremoloSecurity/openunison-k8s-operator/blob/a502b3cd915ab13b6527d5caddf3a619c3b485e8/src/main/js/deploy-upstream-k8s.js#L181

In order to trigger Ingress re-creation it is required remove and redeploy whole _openunison-orchestra_ deployment

**Proposal** : check if ingress is absent , then create it.

**Comment by mlbiam** [comment]
@o-grigorev we have deprecated having the operator create `Ingress` objects (and everything else except the `Secret` objects) in lue of having the helm charts generate these objects since helm is better then our code will be to solve for these sorts of situations.  You can upgrade to these new charts using the steps at https://openunison.github.io/upgrading/

**Comment by o-grigorev** [comment]
I forgot to mention I meant  [openunison-k8s-login-oidc](https://github.com/OpenUnison/helm-charts/tree/master/openunison-k8s-login-oidc) where Ingress is not present in the chart

**Comment by mlbiam** [comment]
Right, we've moved from login specific repos and charts to a single unified repo and chart (https://openunison.github.io/).  This makes it much easier to manage and maintain.  When we did this we also moved all of the object creation (exception for `Secret`s) to the charts.  You can still use your existing values.yaml and `Secret`s, so the move should be transparent to users (other then the few minutes of downtime that's needed to switch).

**Comment by o-grigorev** [comment]
Got it, thanks
